examples/lightrag_openai_demo_query_only.py

```
@@ -0,0 +1,59 @@
+import asyncio
+import argparse
+from lightrag.utils import logger
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)